### DISCUSSION TITLE
Renaming Grub Menu Entries

### DISCUSSION THREAD

**Discussion by niccolopogi** [SpiralLinux/SpiralLinux-project#518] [Q&A] [answered]
Hi. I've been trying out Spiral Linux for the past few weeks in a VM and I'm hoping I can finally install it on my main desktop with this stupid question.

Is it possible to remove the "(Spiral Linux) GNU/Linux" part in the Grub entries for the default boot and the ones with the kernel options? I'm a bit neurotic with this stuff so I'm hoping I can just have "Debian" or "Debian GNU/Linux" as the boot option label. And hopefully with the snapshots too?

I tried a bunch of stuff searching the internet, but I can't seem to make it work via Grub Customizer or editing the config file (I borked grub during that time and had to do a clean reinstall in the VM). Some posts I saw also say to just avoid using Grub Customizer and just remove it.

I hope this doesn't come as offensive and that I'm erasing your fantastic work, just a bit of OCD on my end.

Cheers and thanks.

**Comment by geckolinux** [comment, marked as answer]
Hi there, no problem, thanks for the question. I believe you can remove the "SpiralLinux" part by editing or removing the line that starts with `GRUB_DISTRIBUTOR` in `/etc/default/grub` and then running `sudo update-grub2`

**Reply by niccolopogi** [reply]
It worked! I think I missed the update grub command on my end when I was trying to do it on my own. However the snapshots option is still labelled as SpiralLinux. Any idea how to change that?

![SpiralLinux snapshots](https://github.com/user-attachments/assets/adb73157-3062-43bb-ae93-3d5a0e21ed02)

**Reply by geckolinux** [reply]
Hi there, that is configurable in `/etc/default/grub-btrfs` with the `GRUB_BTRFS_SUBMENUNAME` option.
